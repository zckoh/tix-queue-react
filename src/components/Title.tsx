import { Container, Nav } from "react-bootstrap";
import { Link } from "react-router-dom";
import "./../styles/Title.css";

const Title = () => {
  return (
    <Container className="Title">
      <h1>
        <Nav.Link as={Link} to="/">
          Help Queue
        </Nav.Link>
      </h1>
    </Container>
  );
};

export default Title;
