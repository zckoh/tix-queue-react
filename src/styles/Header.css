.Header {
    border-bottom: 1px solid white;
}