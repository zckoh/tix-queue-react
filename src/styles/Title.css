/*  src/Title.css  */
.Title {
  text-align: center;
  justify-content: center;
  background-color: #282c34;
  color: #fff;
}

.Title a {
  color: #fff;
}
