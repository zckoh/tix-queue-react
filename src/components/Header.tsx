import { Container } from "react-bootstrap";

import Title from "./Title";
import "../styles/Header.css";

const Header = () => {
  return (
    <Container as="header" className="Header py-3">
      <Title />
    </Container>
  );
};

export default Header;
