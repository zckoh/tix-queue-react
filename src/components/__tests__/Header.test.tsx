import { shallow } from "enzyme";

import Header from "../Header";
import Title from "../Title";

describe("<Header />", () => {
  it("contains a <Title/> component", () => {
    const wrapper = shallow(<Header />);
    expect(wrapper.contains(<Title />)).toEqual(true);
    expect(wrapper).toMatchSnapshot();
  });
});
