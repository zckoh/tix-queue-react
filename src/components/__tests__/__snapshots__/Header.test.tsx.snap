// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> contains a <Title/> component 1`] = `ShallowWrapper {}`;
